src/TTGO_T-Beam_LoRa_APRS.ino

```diff
@@ -3224,6 +3224,9 @@ void setup()
   // for diagnostics
   uint32_t t_setup_entered = millis();
 
+  // initialize ESP32 Process WDT, 120s T/O
+  esp_task_wdt_init(120, true);
```

Comment:
You moved it 300 lines up, to begin of setup.
I had it in line 3511, directly followed by esp_task_wdt_add(). I think this is cleaner because they belong to eachother.
In setup(), there's no really need for handling watchdog eventy; that's why I decided to put it right at the end of setup().